tests/integration/test_lists/test-db/l0_dgx_b200.yml

```
@@ -0,0 +1,74 @@
+version: 0.0.1
+l0_dgx_b200:
+- condition:
+    ranges:
+      system_gpu_count:
+        gte: 4
+        lte: 4
+    wildcards:
+      gpu:
+      - '*b200*'
```

I think this might match gb200 if we use a leading wildcard.

```suggestion
      - 'b200*'
```

---

Alternatively, we could add a  `cpu: x86_64` condition

---

The pipeline is using "B200" for the GPU value: 

> trt-test-db -d /tmp/jenkins/workspace/LLM/main/L0_Test-x86_64/llmVanilla/TensorRT-LLM/src/tests/integration/test_lists/test-db --context l0_dgx_b200 --test-names --output /tmp/jenkins/workspace/LLM/main/L0_Test-x86_64/llmVanilla/TensorRT-LLM/src/l0_dgx_b200.txt --match '{"gpu":"B200","gpu_memory":"183359.0","host_mem_available_mib":"2008236.5","host_mem_total_mib":"2063917.33","hostname":"87e16ed4428e","is_aarch64":false,"is_linux":true,"is_native_windows":false,"is_power":false,"is_wsl":false,"system_gpu_count":"4","linux_distribution_name":"ubuntu","linux_version":"24.04","linux_codename":"noble","compute_capability":"10.0","supports_tf32":true,"supports_int8":true,"supports_fp8":true,"cpu":"x86_64","sysname":"Linux","cuda_version":null,"cublas_version":null,"cudnn_version":null,"is_android":null,"is_remote":false,"stage":"post_merge","backend":"pytorch","auto_trigger":"others"}'

You're right though, since `--context l0_dgx_b200` is used here the GB200 pipeline won't access this file.

---

I think we don't need to pass additional parameters to Slurm. The makos are generated by trt-test-db, which is a tool running on the acquired node.

https://github.com/NVIDIA/TensorRT-LLM/blob/main/jenkins/L0_Test.groovy#L799

---

My bad, just added cpu condition for both gb200 and b200